doc/classes/AudioStream.xml

```diff
@@ -45,6 +45,12 @@
 				Override this method to customize the name assigned to this audio stream. Unused by the engine.
 			</description>
 		</method>
+		<method name="_get_tags" qualifiers="virtual const">
+			<return type="Dictionary" />
+			<description>
+				Override this method to customize the tags for this audio stream.
```

Comment:
I would recommend elaborating on what this should return, and in what format, ideally. I can suggest later.

Reply:
Would it be beneficial to list some common tags like "artist", "title" and "album"?